apps/frontend/src/pages/room/components/location/LocationListSection.tsx

```
@@ -242,88 +241,91 @@ export const LocationListSection = ({
+                          )}
+                        </div>
 
-                      <div className="flex-1 flex flex-col justify-between py-0.5">
-                        <div className="flex flex-col gap-1 cursor-pointer" onClick={() => handlePlaceSelect(place)}>
-                          <h3 className="font-bold text-gray-800 text-base line-clamp-1">{place.displayName.text}</h3>
-                          <div className="flex items-center gap-2">
-                            {place.rating && (
-                              <span className="text-xs text-yellow-500 flex items-center gap-0.5">
-                                ★ {place.rating.toFixed(1)}
-                                {place.userRatingCount && <span className="text-gray-400">({place.userRatingCount})</span>}
+                        <div className="flex-1 flex flex-col justify-between py-0.5">
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)